### ISSUE TITLE
The bots are really bugged!

### ISSUE THREAD

**Issue by virgo77** [ZhengPeiRu21/mod-playerbots#127]
I try to invoke a unique bot which is a character of my owned account.
It is as same level as me: 45
We are in party but:
- if I try to take a fly, the bot is not taking it. It's mounting and... bye bye! And when I connect on this 'bot' character, I fall totally outside of the map. A very bad falling, and I don't see the decor anymore :( When I click on 'respawn', I'm just alive, at an another place in the same map (here, at Desolace). Not at the Graveyard. Maybe because my corpse is not reachable
- if I want to take a boat, the bot is like 'afraid', and runs away. It didn't get in the boat. Why? but why? Just follow me hou hou hou
- When I take my mount, sometimes, the bot tries to invoke his mount (a Traveler's Toundra Mammoth) but it fails, 4 times. Then he invokes the only mount I bought for him, and sometimes it does not follow me. We are still in party. But nothing. I have to remove the bot then recall him and, maybe it follows me
- in the picture, I'm at Ratchet. We can see 2 bots, on fly mount near the Flight Master, but not moving since hours ! They are level 6 and 7

I'm trying for other stuff

Ha really, very funny these bots ^^

![WoWScrnShot_101922_001029](https://user-images.githubusercontent.com/23108339/196555018-2c4d45b1-31ad-49ab-ac96-ad7579ab2683.jpg)

**Comment by drozco** [comment]
Yeah they don't follow well.  You can whisper 'summon' to them when you get where you're going.

**Comment by corker22** [comment]
make a macro for summon mine is /p summon. they do ok not great but ok. people are working on it. if you have suggestions on how to make them work it would be helpful. even if you are wrong it may give the people coding it new and fresh ideas. im not one coding this im still trying to read through and catch up on coding after 20 years of not coding
